packages/cozy-client/src/CozyClient.js

```diff
@@ -508,7 +509,12 @@ class CozyClient {
 
     responseDocs.forEach(doc => {
       return forEach(relationshipsByName, (relationship, relName) => {
-        const queryDef = relationship.type.query(doc, this, relationship)
+        const queryDef = relationship.type.query(
+          doc,
+          this,
+          relationship,
+          cursor
+        )
```

Comment:
I feel a bit uneasy here, I am not sure that having `cursor` at this point is necessary.